src/presenter/event-board-presenter.js

```diff
@@ -3,21 +3,72 @@ import EventView from '../view/event-view';
 import EventEditView from '../view/event-edit-view';
 import SortView from '../view/sort-view';
 import {render} from '../render.js';
+import {isEscEvent} from '../utils.js';
 
 export default class BoardPresenter {
-  eventsListComponent = new EventsListView();
+  #boardContainer = null;
+  #eventsModel = null;
+
+  #eventsListComponent = new EventsListView();
+
+  #boardEvents = [];
 
   init = (boardContainer, eventsModel) => {
-    this.boardContainer = boardContainer;
-    this.eventsModel = eventsModel;
-    this.boardEvents = [...this.eventsModel.getEvents()];
+    this.#boardContainer = boardContainer;
+    this.#eventsModel = eventsModel;
+    this.#boardEvents = [...this.#eventsModel.events];
 
-    render(new SortView(), this.boardContainer);
-    render(this.eventsListComponent, this.boardContainer);
-    render(new EventEditView(this.boardEvents[0]), this.eventsListComponent.getElement());
+    render(new SortView(), this.#boardContainer);
+    render(this.#eventsListComponent, this.#boardContainer);
+    // render(new EventEditView(this.#boardEvents[0]), this.#eventsListComponent.element);
 
-    for (const event of this.boardEvents) {
-      render(new EventView(event), this.eventsListComponent.getElement());
+    for (const event of this.#boardEvents) {
+      this.#renderEvent(event);
```

Comment:
Здесь лучше уже foreach использовать
```suggestion
     this.#boardEvents.forEach(this.#renderEvent)
```